code/datums/outfits/outfit.dm

```diff
@@ -26,7 +26,7 @@
 	var/list/cybernetic_implants = null
 
 
-/datum/outfit/proc/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+/datum/outfit/proc/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE, alt_title)
```

Comment:
Can you try to move this behavior to /datum/outfit/job instead of /datum/outfit?